### ISSUE TITLE
no audio file when recording with `mirv_streams` in CS2

### ISSUE THREAD

**Issue by ChetdeJong** [ripieces/advancedfx#990]
Could be related to https://github.com/advancedfx/advancedfx/issues/988

**Comment by hjbdev** [comment]
I think this is still an issue on 2.172.2.0

I'm not getting audio output with `mirv_streams record screen` and 
`mirv_streams settings add ffmpeg n2 \"-c:v h264_nvenc -g 300 -preset slow -tune hq -rc constqp -qp 12 {QUOTE}{AFX_STREAM_PATH}.mkv{QUOTE}\"`

**Comment by dtugend** [comment]
TLDR: Doesn't affect normal movie makers, only if you use `-tools` command line option.

----

> I think this is still an issue on 2.172.2.0
> 
> I'm not getting audio output with `mirv_streams record screen` and `mirv_streams settings add ffmpeg n2 \"-c:v h264_nvenc -g 300 -preset slow -tune hq -rc constqp -qp 12 {QUOTE}{AFX_STREAM_PATH}.mkv{QUOTE}\"`

As we discovered on Discord this is due to VCONSOLE usage (required for netconport) and a bug with it making the engine think that paths are read-only, causing startmovie for the audio used by HLAE to fail.

> Blocked change to read-only path C:\Program Files (x86)\Steam\steamapps\common\Counter-Strike Global Offensive\game\csgo\movie\2024_10_03_18_07_37!
Blocked change to read-only path C:\Program Files (x86)\Steam\steamapps\common\Counter-Strike Global Offensive\game\csgo\movie\2024_10_03_18_07_37\test.wav!

As a workaround you could use mirv-scripts web-socket capability instead of netconport, it's pretty robust if used as designed in the example see mirv-script in misc folder for an example.
